tests/extensions/test_files_store.py

```diff
@@ -58,9 +60,10 @@ def test_spider_closed_even(caplog):
     spider = spider_with_files_store('22')
 
     extension = FilesStore.from_crawler(spider.crawler)
+    item = spider.build_file_from_response(response_fixture(), file_name='file.json', data_type='release_package')
+    extension.item_scraped(item, spider)
     with caplog.at_level(logging.INFO):
-        extension.spider_closed(spider)
-
+        extension.spider_closed(spider, 'finished')
```

Comment:
Keep the newline between the test setup and the assertions.

```suggestion
        extension.spider_closed(spider, 'finished')

```